ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -484,13 +484,19 @@ private void printTopicInfo(final SourceDescription source) {
     writer().println(String.format("%-20s : %s", "Value format", source.getFormat()));
 
     if (!source.getTopic().isEmpty()) {
-      writer().println(String.format(
-          "%-20s : %s (partitions: %d, replication: %d)",
+      String topicInformation = String.format("%-20s : %s",
           "Kafka topic",
-          source.getTopic(),
-          source.getPartitions(),
-          source.getReplication()
-      ));
+          source.getTopic()
+      );
+
+      if (source.getPartitions() != 0) {
```

Comment:
I don't love this check but I can't think of a better suggestion offhand. Would be curious to know what others think. 

At minimum if we're sticking with this, we should add a comment explaining why this could happen/the thinking behind having this check so future readers of the code aren't confused.